src/App.js

```diff
@@ -1,9 +1,68 @@
 import React from 'react';
 import './App.css';
+import Buttons from "./components/Buttons";
+import Products from "./components/Products";
+import Tiles from "./components/Tiles";
+import bag1 from "./assets/bag_1.png"
+import bag2 from "./assets/bag_2.png"
+import bag3 from "./assets/bag_3.png"
+import bag4 from "./assets/bag_4.png"
+import brand from "./assets/brand.png"
+import story from "./assets/our_story.png"
 
 function App() {
+
   return (
-      <h1>Handbags & Purses</h1>
+      <>
+        <nav>
+          <h1>Handbags & Purses</h1>
+          <Buttons knopje="button" waarde={false} naam="to the collection"/>
+          <Buttons knopje="button" waarde={false} naam="shop all bags"/>
+          <Buttons knopje="button" waarde={true} naam="pre-orders"/>
```

Comment:
Ik begrijp dat het makkelijk is om Nederlandse woorden te gebruiken voor de benamingen van properties maar het is gebruikelijk om hier Engelse benamingen  voor te gebruiken.

In de opdracht stond dat de buttontekst in de console log moest verschijnen. Dit gebeurd niet.